aws/resource_aws_api_gateway_rest_api_test.go

```diff
@@ -1176,13 +1310,96 @@ resource "aws_api_gateway_rest_api" "test" {
 `, rName, endpointType)
 }
 
-func testAccAWSAPIGatewayRestAPIConfig_DisableExecuteApiEndpoint(rName string, disabled bool) string {
+func testAccAWSAPIGatewayRestAPIConfigDisableExecuteApiEndpoint(rName string, disableExecuteApiEndpoint bool) string {
+	return fmt.Sprintf(`
+resource "aws_api_gateway_rest_api" "test" {
+  disable_execute_api_endpoint = %[2]t
+  name                         = %[1]q
+}
+`, rName, disableExecuteApiEndpoint)
+}
+
+func testAccAWSAPIGatewayRestAPIConfigDisableExecuteApiEndpointOverrideBody(rName string, configDisableExecuteApiEndpoint bool, bodyDisableExecuteApiEndpoint bool) string {
+	return fmt.Sprintf(`
+resource "aws_api_gateway_rest_api" "test" {
+  disable_execute_api_endpoint = %[2]t
+  name                         = %[1]q
```

Comment:
```suggestion
  name                         = %[1]q
  disable_execute_api_endpoint = %[2]t
```
Super nit

Reply:
Most of the newer test resource configurations should be alphabetical for human readability, it just so happens that there is also the potentially competing convention of `rName` always being the first `fmt.Sprintf()` parameter. 🙁  This should go away if/when a newer testing framework is available that works with Terraform variables instead. We could that style of configuration now, but since we don't know the final design of that framework, I would be hesitant to create yet another style of configuration in the meantime.